dapp/src/components/Searchbox.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+'use client';
+
+import { ListItem, Search, SearchBarType, Suggestion } from '@iota/apps-ui-kit';
+import { useState } from 'react';
+
+export default function SearchBox() {
+    const [searchValue, setSearchValue] = useState('');
+    const [filteredSuggestions, setFilteredSuggestions] = useState<Suggestion[]>([]);
+
+    const allSuggestions: Suggestion[] = [
+        { id: '1', label: 'Daiki', supportingText: 'Caption' },
+        { id: '2', label: 'Iota' },
+        { id: '3', label: 'Pepe' },
+        { id: '4', label: 'Gon' },
+    ];
+
+    const handleSearchValueChange = (value: string) => {
+        setSearchValue(value);
+        const filtered =
+            allSuggestions.filter((suggestion) =>
+                suggestion.label.toLowerCase().includes(value.toLowerCase()),
+            ) || [];
+        setFilteredSuggestions(filtered);
+    };
+
+    const handleSuggestionClick = (suggestion: Suggestion) => {
+        setSearchValue(suggestion.label);
+        setFilteredSuggestions([]);
+    };
+
+    return (
+        <div className="max-w-[360px] h-full w-full">
```

Comment:
manage the width from the parent
```suggestion
        <div className="w-full">
```